benchmarks/zlib_zlib_uncompress_fuzzer/compress_fuzzer.c

```diff
@@ -0,0 +1,81 @@
+#include <stdio.h>
```

Comment:
A problem with copying oss-fuzz/projects/$PROJECT dir into fuzzbench is a lot of the files will be unnecesary. What should be the policy on this? Just copy all files or just the ones needed for the benchmark (this benchmark isn't using compress_fuzzer to be clear). A problem with removing unneeded files is some build.sh files might depend on them. I think forcing removal might be more complicated than it is worth.

Reply:
If we're copying the files over, then we i think we should definitely not copy unnecessary files, and should fix the build.sh. By copying these files over, these files will have a separate, independent life in fuzzbench.